applications/IgaApplication/tests/NodeSurfaceGeometry3DTests.py

```diff
@@ -0,0 +1,142 @@
+from KratosMultiphysics import *
+from KratosMultiphysics.IgaApplication import *
+
+import KratosMultiphysics.KratosUnittest as KratosUnittest
+
+
+def GetFilePath(fileName):
+    return os.path.dirname(__file__) + "/" + fileName
+
+
+class NodeSurfaceGeometry3DTests(KratosUnittest.TestCase):
+
+    def setUp(self):
+        # generate nodes = control points
+        node_00 = Node(1, 0.0, 0.0, 0.0)
```

Comment:
This node is from you application or the one in the core?

Reply:
Core

Reply:
Ok, there was a Node.h instead of node.h before, it is that correct?